csrc/scheduler/resize.cpp

```diff
@@ -133,6 +120,30 @@ bool ResizeScheduler::canScheduleCompileTime(Fusion* fusion) {
     return false;
   }
 
+  for (auto out_tv : ir_utils::filterByType<TensorView>(fusion->outputs())) {
```

Comment:
This check is needed since the non-exclusivity check is dropped. It was redundant before.